### PR TITLE
Additions to the repo

### PR DESCRIPTION
**Pull request by shortc1rcuit**

### CODE REVIEW

File: HTML/Shapes And Bombs interactive.html [hunk 1]
```diff
@@ -441,28 +443,28 @@ <h3>Step 3 (Shape):</h3>
...
+            <div class="page-content">
                 <ul>
                     <li>If the module’s initial letter value <span class="bold">(Table 1)</span> is even, only count in the upper half; otherwise, only count in the bottom half.</li>
                     <li>Next, look up the corresponding table to see which shape fits the number you got:</li>
                     <li>If the number is smaller than 5, leave it at 5, otherwise; if it is greater than 14, leave it at 14.</li>
                     <li><span class="bold">NOTE:</span> White squares means unlit and gray squares means lit.</li>
                 </ul>
 
-                <table align = "center" class = "table-empty">
+                <table align="center" class="table-empty">
```
Comment (kavinkul) [inline HTML/Shapes And Bombs interactive.html:464]: `align` is deprecated.

File: HTML/Cookie Clicker.html [hunk 2]
```diff
@@ -1,5 +1,7 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
+<!-- saved from url=(0049)https://ktane.timwi.de/HTML/Cookie%20Clicker.html -->
 <html>
+<link type="text/css" id="dark-mode" rel="stylesheet" href="https://ktane.timwi.de/HTML/Cookie%20Clicker.html">
```
Comment (kavinkul) [inline HTML/Cookie Clicker.html:4]: This line should probably be removed or if it has any importance then it should be in `<head>`

File: HTML/Cheat Checkout.html [hunk 3]
```diff
@@ -1,628 +1,668 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
 <html>
 <head>
     <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <meta charset="utf-8">
```
Comment (sleepin-fang) [inline HTML/Cheat Checkout.html:5]: Repetitive. Can be deleted

File: HTML/Cookie Clicker.html (hunk 2 above)
Comment (Timwi) [reply HTML/Cookie Clicker.html:4]: This line is indeed quite invalid, as it says it links to a CSS style sheet but the link is actually to an HTML page

**Pull request merged**